src/index.tsx

```diff
@@ -43,7 +43,7 @@ const router = createBrowserRouter(
 const root = ReactDOM.createRoot(document.getElementById('root') as HTMLElement)
 root.render(
     <React.StrictMode>
-        <AlertsProvider>
+        <AlertsProvider plugin={false} parentAlertsAdd={() => { }} showAlertsInPlugin={false}>
```

Comment:
Is this a fix for [this](https://github.com/dhis2-sre/im-web-client/pull/377)?

Reply:
yes it was an attempt to fix it, seems to stop build from passing, will have to remove it.